config/prow/plugins.yaml

```diff
@@ -491,67 +491,67 @@ repo_milestone:
     # You can curl the following endpoint in order to determine the github ID of your team
     # responsible for maintaining the milestones. You may need to specify the page number.
     # curl -H "Authorization: token <token>" "https://api.github.com/orgs/<org-name>/teams?page=N"
-    maintainers_id: 2460384
+    # maintainers_id: 2460384
```

Comment:
Nit: I think we could just clean all this up now, including this comment about the IDs. Sounds like IDs are no longer relevant and the team name can just be used directly going forward.

Reply:
done